libs/beacon/tests/dkg/beacon_manager_tests.cpp

```diff
@@ -30,16 +30,16 @@ using Certificate    = Prover;
 using CertificatePtr = std::shared_ptr<Certificate>;
 using MuddleAddress  = byte_array::ConstByteArray;
 
-struct DkgOutput
-{
-  PrivateKey              private_key;
-  PublicKey               group_public_key;
-  std::vector<PublicKey>  public_key_shares;
-  std::set<MuddleAddress> qual;
-};
-
 TEST(beacon_manager, dkg_and_threshold_signing)
 {
+  struct DkgOutput
```

Comment:
Seems like it makes sense to have this struct a part of the codebase and Get/SetDkgOutput function signature is just DkgOutput GetDkgOutput(); ?

Reply:
As I mention below, the DkgOutput here is different from the one declared in mcl_dkg as qual is included here.